test/libsolidity/semanticTests/salted_create/prediction_example.sol

```diff
@@ -0,0 +1,29 @@
+contract D {
+    uint public x;
+    constructor(uint a) {
+        x = a;
+    }
+}
+
+contract C {
+    function createDSalted(bytes32 salt, uint arg) public {
+        address predictedAddress = address(uint160(uint(keccak256(abi.encodePacked(
+            bytes1(0xff),
+            address(this),
+            salt,
+            keccak256(abi.encodePacked(
+                type(D).creationCode,
+                arg
+            ))
+        )))));
+
+        D d = new D{salt: salt}(arg);
+        require(address(d) == predictedAddress, "Address mismatch.");
+    }
+}
+// ====
+// EVMVersion: >=constantinople
+// compileViaYul: also
```

Comment:
Actually, we don't need to set `compileViaYul: also` explicitly anymore. This is now the default. I think the issue comes from before the time when we did it.

Reply:
I just copy pasted from the issue 😅

Do you want it to be changed?

Reply:
I know, not complaining, just pointing it out :)

I guess would be better to change it but it's very minor so feel free to also just merge it if you don't want to bother.

Reply:
Would be nice to avoid another 1hr CI run.